src/Aspire.Hosting.Dapr/DaprOptions.cs

```diff
@@ -11,13 +11,13 @@ public sealed record DaprOptions
     /// <summary>
     /// Gets or sets the path to the Dapr CLI.
     /// </summary>
-    public string? DaprPath { get; init; }
+    public string? DaprPath { get; set; }
```

Comment:
If it is "`init`", it is impossible to set these options via the new `AddDapr()` callback-based interface.

Reply:
Also, why is DaprOptions a record?

Reply:
Where's the new `AddDapr()` interface?